### DISCUSSION TITLE
parsing 4/5 axis cam data to robots

### DISCUSSION THREAD

**Discussion by lin-ycv** [visose/Robots#301] [Q&A] [answered]
I'm looking for CAM programs to support a robot milling project, I've seen a few potential candidate, but none will support a robot directly. most will output a 4/5 axis toolpath.

anyone have experience converting 4/5 axis CAM data to robots?

**Comment by visose** [comment, marked as answer]
The Extensions plugin contains components to convert a g-code file to a list of Robots targets.
You will need to create a custom postprocessor in the CAM app that generates g-code in the format expected by the component.
